### PR TITLE
Generalise wkt normalisation boundary check for any boundary

### PR DESCRIPTION
**Pull request by CarlosCoelhoSL**
Trello ticket: https://trello.com/c/tyDiJ0yi/453-investigate-and-detect-brownfield-land-sites-that-fall-outside-their-lpa-geometry

# The problem

In order to detect whether brownfield land is in the wrong place, there must be a process that can accept an LPA boundary and check whether the point lies within that boundary. Currently, the normalisation process in the wkt class verifies that the wkt lies within a hardcoded bounding box of England, so this can be modified to accept a custom boundary.

# What's changed

This PR adds an optional input to the normalisation function , `boundary`, which then is used to validate that the wkt lies within this custom boundary. The boundary must of a wkt of type Polygon or MultiPolygon. If no boundary is input into the function, it will use a default boundary, this being the 'within England' bounding box.

Notes:

While investigating this area of the code, it was found that the 'within England' bounding box used when judging WGS84 and OSGB crs were different from each other. In order to create a unified idea of 'within England', the default boundary used here is the intersection of the two original bounding boxes

### CODE REVIEW

**Review by GeorgeGoodall — APPROVED** [review]
LGTM

**Review by eveleighoj — CHANGES_REQUESTED** [review]

**Comment by hyperoceanic** [comment]
We should ensure we update the documentation materials when we change the application logic.

**Review by morgan-sl — APPROVED** [review]
code looks ok.

File: digital_land/datatype/wkt.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import shapely.wkt
 import json
 import logging
-from shapely.geometry import shape
+from shapely.geometry import shape, Point
 from shapely.errors import WKTReadingError
 from shapely.ops import transform
 from shapely.geometry import MultiPolygon
```
[the remark this answers is not in the document]
Comment (eveleighoj) [reply digital_land/datatype/wkt.py:7]: Looks good just update it with the bounds after the NSIP stuff.

**Pull request merged**